### PR TITLE
[Merged by Bors] - refactor(Topology/Order): Rename order topologies

### PR DESCRIPTION
**Pull request by mans0954**
Rename the Upper/Lower UpperSet/LowerSet topologies to the conventions suggested by @YaelDillies in https://github.com/leanprover-community/mathlib4/pull/2508#discussion_r1306711450

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — APPROVED** [review]
Your Ctrl+F was a bit overzealous, so I fixed it.

maintainer merge

File: Mathlib/Topology/Order/LowerUpperTopology.lean [hunk 1]
```diff
@@ -20,20 +20,20 @@ right-closed left-infinite intervals.
 
 ## Main statements
 
-- `LowerTopology.t0Space` - the lower topology on a partial order is T₀
-- `LowerTopology.isTopologicalBasis` - the complements of the upper closures of finite
+- `IsLower.t0Space` - the lower topology on a partial order is T₀
+- `IsLower.isTopologicalBasis` - the complements of the upper closures of finite
```
Comment (ADedecker) [inline Mathlib/Topology/Order/LowerUpperTopology.lean:24]:
```suggestion
- `Topology.IsLower.t0Space` - the lower topology on a partial order is T₀
- `Topology.IsLower.isTopologicalBasis` - the complements of the upper closures of finite
```

File: Mathlib/Topology/Order/LowerUpperTopology.lean [hunk 2]
```diff
@@ -20,20 +20,20 @@ right-closed left-infinite intervals.
...
   subsets form a basis for the lower topology
-- `LowerTopology.continuousInf` - the inf map is continuous with respect to the lower topology
+- `IsLower.continuousInf` - the inf map is continuous with respect to the lower topology
```
Comment (ADedecker) [inline Mathlib/Topology/Order/LowerUpperTopology.lean:26]:
```suggestion
- `Topology.IsLower.continuousInf` - the inf map is continuous with respect to the lower topology
```

**Review by ADedecker — COMMENTED** [review]
Thanks!

bors d+

File: Mathlib/Topology/Order/LowerUpperTopology.lean (hunk 1 above)
Comment (mans0954) [reply Mathlib/Topology/Order/LowerUpperTopology.lean:24]: I think that's what I had originally https://github.com/leanprover-community/mathlib4/pull/7310/commits/c3c74570a0dac68377b95f55e40a66a068e08a1a#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386R23 but @YaelDillies changed it back:

https://github.com/leanprover-community/mathlib4/pull/7310/commits/6ec78992b4f30a3441a210a37139c8409fe5fde7#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386R23

File: Mathlib/Topology/Order/LowerUpperTopology.lean (hunk 2 above)
Comment (mans0954) [reply Mathlib/Topology/Order/LowerUpperTopology.lean:26]: Again, I think I had that https://github.com/leanprover-community/mathlib4/commit/c3c74570a0dac68377b95f55e40a66a068e08a1a#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386L26 but @YaelDillies changed it to https://github.com/leanprover-community/mathlib4/commit/6ec78992b4f30a3441a210a37139c8409fe5fde7#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386R26

File: Mathlib/Topology/Order/LowerUpperTopology.lean (hunk 1 above)
Comment (ADedecker) [reply Mathlib/Topology/Order/LowerUpperTopology.lean:24]: Ah right, I think `doc-gen` is now a bit better at linking not-fully-qualified names, so you can keep it like that. You can ignore these comments then.

**Pull request closed without merging**